src/client/client.go

```diff
@@ -0,0 +1,39 @@
+package client
```

Comment:
Why is this package named "client" if its only function is to get kubectl config?